python/lancedb/query.py

```diff
@@ -480,6 +481,12 @@ def to_arrow(self) -> pa.Table:
 
         # get the index path
         index_path = self._table._get_fts_index_path()
+        # check if the index exist
+        if not Path(index_path).exists():
+            raise FileNotFoundError(
+                "Fts index does not exist."
+                f"Please first call table.create_fts_index('<field_names>') to create the fts index."
```

Comment:
```suggestion
                f"Please first call table.create_fts_index(['<field_names>']) to create the fts index."
```
nit picky but just want to make it clear that when you have multiple fields, you pass in a list rather than concatenate field names in a string